.travis.yml

```diff
@@ -1,34 +1,31 @@
 language: c
 env:
-  - BD=esp8266:esp8266:d1_mini:CpuFrequency=80,FlashSize=4M3M
+  - BD=esp8266:esp8266:d1_mini:xtal=80,eesz=4M3M
```

Comment:
I have just tried this out with with CPU frequency set to 160, file upload speed has more than doubled. Would it be an idea to change this to 160 ?